templates/supplier/suppliers.html

```diff
@@ -0,0 +1,27 @@
+{% load static %}
+<!DOCTYPE html>
+<html>
+
+<head>
+	<title>Suppliers</title>
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.1/css/all.css" integrity="sha384-gfdkjb5BdAXd+lj+gudLWI+BXq4IuLW5IT+brZEZsLFm++aCMlF1V92rMkPaX4PP" crossorigin="anonymous">
+	<link rel="stylesheet" href="/static/css/suppliers.css">
+</head>
+
+<body>
+    {% include 'buy_together_app/help_parts/nav.html' %}
+
+	<div class="container mt-5 p-3 align-center">
+		<div class="container_A d-flex justify-content-center">
+			<p> here will be something</p>
+		</div>
+		<div class="container_A d-flex justify-content-center">
+			<p> here will be something</p>
```

Comment:
?

Reply:
the goal was so create PR that includes the basic template for supplier so others can build their PRs on top of this one.

Reply:
alright